.github/workflows/build_rust_docs.yaml

```diff
@@ -67,9 +70,12 @@ jobs:
         run: |
           cd ./out
           git add .
+          echo added
```

Comment:
Did you forget these from debugging or is it intentional?

Reply:
forgot them :) good catch